components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/model/ActionInvocationSuccessResponse.java

```diff
@@ -75,6 +82,13 @@ public Builder operations(@JsonProperty("operations") List<PerformableOperation>
             return this;
         }
 
+        @JsonProperty("data")
+        public Builder data(@JsonProperty("data") ExtendedData data) {
+
+            this.data = data;
+            return this;
+        }
+
         public ActionInvocationSuccessResponse build() {
```

Comment:
Isn't the builder now expecting operations to be not null ?
With this it should be either operations or data that should be present